Design/ISIS_Imaging/ISIS_Imaging_general.md

```diff
@@ -1,61 +1,112 @@
+    - [9.10. Applying a filter](#910-applying-a-filter)
+        - [9.10.1. Dynamic dialogue building](#9101-dynamic-dialogue-building)
+        - [9.10.2. Transferring information from the dialogue (the parameters) to the execution](#9102-transferring-information-from-the-dialogue-the-parameters-to-the-execution)
+    - [9.11. Preview a result](#911-preview-a-result)
+    - [9.12. Undoing an operation](#912-undoing-an-operation)
+    - [9.13. Process List](#913-process-list)
+        - [9.13.1. Store image state as part of the History in the process list](#9131-store-image-state-as-part-of-the-history-in-the-process-list)
+        - [9.13.2. Exporting of Process List](#9132-exporting-of-process-list)
+    - [9.14. Processing a full volume](#914-processing-a-full-volume)
+    - [9.15. Issues with Finding Center of Rotation and Reconstruction](#915-issues-with-finding-center-of-rotation-and-reconstruction)
+        - [9.15.1. Automatic Center of Rotation (COR) with imopr cor](#9151-automatic-center-of-rotation-cor-with-imopr-cor)
+        - [9.15.2. Center of Rotation (COR) with imopr corwrite](#9152-center-of-rotation-cor-with-imopr-corwrite)
+        - [9.15.3. Tilt correction](#9153-tilt-correction)
+        - [9.15.4. Visualising the tilt](#9154-visualising-the-tilt)
+        - [9.15.5. Calculating the real tilt angle](#9155-calculating-the-real-tilt-angle)
+    - [9.16. Reconstruction](#916-reconstruction)
+    - [9.17. Tools and Algorithms](#917-tools-and-algorithms)
+    - [9.18. Remote submission and MPI-like behaviour](#918-remote-submission-and-mpi-like-behaviour)
+        - [9.18.1. Remote compute resource used at ISIS: SCARF](#9181-remote-compute-resource-used-at-isis-scarf)
+        - [9.18.2. MPI-like behaviour](#9182-mpi-like-behaviour)
 
 <!-- /TOC -->
 
 The user requirements for the interface have been listed [here](https://github.com/mantidproject/isis_imaging/wiki/High-Level-User-Requirements-and-Use-Cases) and have been reviewed and confirmed by the imaging scientists.
 
 The development requirements, following from the user requirements, are listed [here](https://github.com/mantidproject/isis_imaging/wiki/High-Level-Development-Requirements-and-Guidelines)
 
-# Motivation
+# 1. Motivation
 
 The current [Tomography Reconstruction](https://github.com/mantidproject/mantid/tree/043095a619bc8851942a31253a1a8f8b820ab30f/MantidQt/CustomInterfaces/inc/MantidQtCustomInterfaces/Tomography) interface was found to not satisfy the current imaging requirements. An easy solution to the problems was not found. With the agreement of the IMAT imaging scientists and senior members of the Mantid team, work is to be started on this design for a rewrite of the interface. Python is the preferred language here, in order to be able to make use of visualising libraries like Matplotlib and interactions with Python based reconstruction packages like Tomopy.
 
-# User Requirements
+# 2. Old Tomography GUI Design
```

Comment:
It seems like this section combines what is missing in the old GUI and some elements of what you want to add to the new GUI. It may be better to separate the two.